tracer/test/test-applications/integrations/LogsInjection.Serilog/Program.cs

```diff
@@ -15,6 +15,42 @@
 
 namespace LogsInjection.Serilog
 {
+    /// <summary>
+    /// Custom LogEventPropertyValue that doesn't match any of the duck types
+    /// (ScalarValueDuck, SequenceValueDuck, StructureValueDuck, DictionaryValueDuck).
+    /// This reproduces the bug where FormatLogEventPropertyValue doesn't write a value
+    /// when debug logging is disabled.
+    /// </summary>
+    public class CustomPropertyValue : LogEventPropertyValue
+    {
+        private readonly object _value;
+
+        public CustomPropertyValue(object value)
+        {
+            _value = value;
+        }
+
+        public override void Render(TextWriter output, string format = null, IFormatProvider formatProvider = null)
```

Comment:
I'm actually wondering if we _should_ be able to handle this 🤔 e.g. why couldn't we duck type `LogEventPropertyValue`, write to `StringBuilder` (using a `StringWriter` as the `TextWriter`), and then write _that_ as a simple string in our `SerilogLogFormatter`? 🤔 

Not necessarily required for the fix, because we don't do that today, but should be possible and not too hard?

Interesting that serilog file writing doesn't handle that though